### PR TITLE
fix: expose line ending

### PR DESCRIPTION
**Pull request by peternhale**
@W-12537051@

### CODE REVIEW

**Review by WillieRuemmele — APPROVED** [review]
seems right to me. Any tests or way to verify it's fixing the bug?

File: src/api/bulk.ts [hunk 1]
```diff
@@ -1330,6 +1330,7 @@ export class IngestJobV2<
           externalIdFieldName: this.jobInfo?.externalIdFieldName,
           object: this.jobInfo?.object,
           operation: this.jobInfo?.operation,
+          lineEnding: this.jobInfo?.lineEnding,
```
Comment (cristiand391) [inline src/api/bulk.ts:1333]: What happens if the caller doesn't specify a line ending? docs says it defaults to `LF` if it's not specified, does the same happens if the field in the body passed `undefined`?

Comment (peternhale) [reply src/api/bulk.ts:1333]: I can default it based on platform

Comment (cristiand391) [reply src/api/bulk.ts]: I'm not sure how webpack handles the `process.env` object so not sure what's needed for it to replace it in the browser bundle. Maybe just default the param to `LF`?

**Comment by peternhale** [comment]
> 

@WillieRuemmele it all get passed through to the backend.

**Review by WillieRuemmele — APPROVED** [review]

File: src/api/bulk.ts (hunk 1 above)
Comment (cromwellryan) [reply src/api/bulk.ts:1337]: @cristiand391 it's using [envify](https://github.com/jsforce/jsforce/blob/master/gulpfile.babel.js#LL106C8-L106C14)

Comment (cristiand391) [reply src/api/bulk.ts:1337]: @cromwellryan that's the `master`/v1 branch, I don't see that in the  `2.0` branch:
https://github.com/jsforce/jsforce/tree/2.0

Comment (cristiand391) [reply src/api/bulk.ts:1337]: We tested this by setting `lineEnding = undefined` and the server ignores it and defaults to `LF` as expected.

**Review by cristiand391 — APPROVED** [review]

**Pull request merged**